src/viewer-selector.js

```diff
@@ -124,6 +125,7 @@ ViewerSelector.prototype.createDialog_ = function(options) {
   s.background = '#fafafa';
   s.fontFamily = "'Roboto', sans-serif";
   s.boxShadow = '0px 5px 20px #666';
+  s["z-index"] = 120;
```

Comment:
can't this just be `2` (see [here])(https://github.com/borismus/webvr-polyfill/blob/a18c4af34f/examples/basic/index.html#L31)?